src/fieri/app/models/contributing_file_worker.rb

```diff
@@ -0,0 +1,66 @@
+require 'sidekiq'
+require 'net/http'
+require 'octokit'
+
+class ContributingFileWorker
+  include ::Sidekiq::Worker
+
+  def perform(cookbook_json)
+    evaluate_result = evaluate(cookbook_json)
+
+    Net::HTTP.post_form(
+      URI.parse("#{ENV['FIERI_SUPERMARKET_ENDPOINT']}/api/v1/quality_metrics/contributing_file_evaluation"),
+      fieri_key: ENV['FIERI_KEY'],
+      cookbook_name: cookbook_name(cookbook_json),
+      contributing_file_failure: evaluate_result,
+      contributing_file_feedback: give_feedback(evaluate_result)
+    )
+  end
+
+  private
+
+  def cookbook_name(cookbook_json)
+    JSON.parse(cookbook_json)['name']
+  end
+
+  def source_user_repo(cookbook_json)
+    # http://rubular.com/r/wHJrs02iLs
+    cookbook_json.match(%r{(?<=github.com\/)[\w-]+\/[\w-]+}).to_s
+  end
+
+  def evaluate(cookbook_json)
+    repo = source_user_repo(cookbook_json)
+
+    if repo.empty?
+      # if no match for repo from #source_user_repo, fails metric
+      return true
+    end
+
+    begin
+      octokit_client.contents(repo, path: 'CONTRIBUTING.md')
```

Comment:
I see that [QM011](https://github.com/chef-cookbooks/cookbook-quality-metrics/blob/f45c46c5a24efc397a2749f16fb8bf5549014ff0/quality-metrics/qm-011-contributing_doc.md) states specifically `CONTRIBUTING.md`, but _must_ a document be in markdown? Maybe that's a question for an adjustment to QM011, so `.md` is OK for this first version of the metric.